app/models/visitor.rb

```diff
@@ -0,0 +1,18 @@
+class Visitor < ActiveRecord::Base
+  include Person
+
+  belongs_to :visit
+  validates :visit, :rank, presence: true
```

Comment:
I wanted to avoid `order` because it's already a keyword in SQL and a special key in Rails.